### PR TITLE
add ownerOnly test cases

### PR DESCRIPTION
**Pull request by daemon1024**
Ref #300 #351 #355

### CODE REVIEW

File: examples/multiubuntu/security-policies/ksp-ubuntu-3-proc-path-owner-allow.yaml [hunk 1]
```diff
@@ -10,23 +10,38 @@ spec:
...
-      recursive: true
-  action:
-    Allow
+      - dir: /etc/ # required to change root to user1 (coarse-grained way)
+        recursive: true
+      - dir: /proc/ # required to change root to user1 (coarse-grained way)
+        recursive: true
+  action: Allow
+---
+apiVersion: security.kubearmor.com/v1
```
Comment (nam-jaehyun) [inline examples/multiubuntu/security-policies/ksp-ubuntu-3-proc-path-owner-allow.yaml:33]: it seems that we don't need the Audit policy

File: examples/multiubuntu/security-policies/ksp-ubuntu-3-proc-path-owner-block.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  selector:
+    matchLabels:
+      container: ubuntu-3
+  process:
+    matchPaths:
+      - path: /home/user1/hello
+        ownerOnly: true
+  action: Block
+---
+apiVersion: security.kubearmor.com/v1
```
Comment (nam-jaehyun) [inline examples/multiubuntu/security-policies/ksp-ubuntu-3-proc-path-owner-block.yaml:17]: it seems that we don't need the Audit policy

File: tests/scenarios/multiubuntu_test_14/cmd1 [hunk 3]
```diff
@@ -0,0 +1,7 @@
+source: ubuntu-3-deployment
+cmd: /home/user1/hello
+result: failed
+---
+operation: Process
+condition: /home/user1/hello
+action: Audit
```
Comment (nam-jaehyun) [inline tests/scenarios/multiubuntu_test_14/cmd1:7]: Audit -> Allow?

File: tests/scenarios/multiubuntu_test_14/cmd2 [hunk 4]
```diff
@@ -0,0 +1,7 @@
+source: ubuntu-3-deployment
+cmd: su - user1 -c /home/user1/hello
+result: passed
+---
+operation: Process
+condition: /home/user1/hello
+action: Audit
```
Comment (nam-jaehyun) [inline tests/scenarios/multiubuntu_test_14/cmd2:7]: Audit -> Allow?

File: tests/scenarios/multiubuntu_test_15/cmd1 [hunk 5]
```diff
@@ -0,0 +1,7 @@
+source: ubuntu-3-deployment
+cmd: /home/user1/hello
+result: failed
+---
+operation: Process
+condition: /home/user1/hello
+action: Audit
```
Comment (nam-jaehyun) [inline tests/scenarios/multiubuntu_test_15/cmd1:7]: Audit -> Block?

File: tests/scenarios/multiubuntu_test_15/cmd2 [hunk 6]
```diff
@@ -0,0 +1,7 @@
+source: ubuntu-3-deployment
+cmd: su - user1 -c /home/user1/hello
+result: passed
+---
+operation: Process
```
Comment (nam-jaehyun) [inline tests/scenarios/multiubuntu_test_15/cmd2:5]: Audit -> Block?

**Comment by daemon1024** [comment]
@nam-jaehyun I used Audit because otherwise I wouldn't get logs for testcase when owner is allowed access.
I based it on testcase 8 where we are checking for allow access but used audit cause we wouldn't get the allowed logs otherwise.

**Comment by nam-jaehyun** [comment]
@daemon1024 that is the intention of the policy. we don't generate alerts for allowed operations.

**Pull request merged**